beetsplug/fetchart.py

```diff
@@ -401,6 +401,23 @@ def get(self, album, plugin, paths):
                                       match=Candidate.MATCH_EXACT)
 
 
+class VGMdb(RemoteArtSource):
+    NAME = "VGMdb"
+    URL = 'https://vgmdb.info/album/%s?format=json'
+    INDICES = (1)
+
+    def get(self, album, plugin, paths):
+        print("SLT")
+        print(album.vgmdb_album_id)
+        if album.vgmdb_album_id:
+            try:
+                resp = self.request(self.URL % album.vgmdb_album_id)
+                url = resp.json()['picture_full']
```

Comment:
I suspect that `KeyError`s from this line should also be caught, in case vgmdb has not artwork for an album? Maybe, you can find an example of an album without artwork and see how this is represented in the JSON?